Cargo.toml

```diff
@@ -9,7 +9,8 @@ members = [
     "common/middleware",
     "common/journald",
     "common/state",
-    "bench"
+    "bench",
```

Comment:
If we drop bench and utils/metrics-recorder from the workspace here we may not need the dead code annotations.

Reply:
I removed the annotations without removing the packages from the workspace and I didn't get any compile errors; perhaps because it's in a separate package, the compiler does't complain.